Resources/Locale/en-US/_Goobstation/changeling/store/changeling-catalog.ftl

```diff
@@ -1,140 +1,125 @@
-
-# combat
+# Combat
 
 evolutionmenu-combat-armblade-name = Arm Blade
 evolutionmenu-combat-armblade-desc =
-    Reform one of your arms into a grotesque blade, composed of bone and flesh, able to pry open airlocks and cut through your foes like butter.
-    Costs 15 chemicals.
+    We recreate one of our arms into a grotesque blade, capable of prying open airlocks and maiming our enemies. Cannot be used in our lesser form.
```

Comment:
reshape > recreate imo. or just "shape" as the verb here. i also wonder if its more beneficial to state what CAN be used in lesser form rather than what can't. if lesser form restricts less than half the abilities in the changeling catalog i think this specific verbage is fine, but if lesser form restricts more than half then you should change course and instead indicate what can be used rather than what can't. holy shit i hope that makes sense i just woke up